packages/harmony/src/icons/AnimatedIcons.stories.mdx

```diff
@@ -1,16 +1,24 @@
 import { Meta, Title } from '@storybook/blocks'
 
 import { Text } from 'components'
+import { Heading } from 'storybook/components'
 
 <Meta title='Icons/Animated Icons' />
 
 <Title>Animated Icons</Title>
 
-### Overview
+<Heading>
+## Overview
 
 Animated icons are for Design team use only.
 
-<Text color='danger'>Do not export icons from this section.</Text>
+<span
+  css={(theme) => ({ p: { color: `${theme.color.status.error} !important` } })}
+>
+  Do not export icons from this section.
+</span>
```

Comment:
was the Text component not working here?

Reply:
Realized for harmony docs we don't want to use text for elements that will be styled with harmony overrides so this just makes that a bit more explicit. Don't love this though, maybe we have a harmony docs text component or something